.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: go
 
 go:
-  - 1.10.x
-  - 1.11.x
+  - 1.11.4
```

Comment:
```suggestion
  - "1.11.4"
```
I think the quotes are necessary now to avoid yaml weirdness.

Reply:
Ah really? Huh.